website/source/docs/agent/configuration/client.html.md

```diff
@@ -133,6 +133,19 @@ see the [drivers documentation](/docs/drivers/index.html).
     }
     ```
 
+- `"driver.blacklist"` `(string: "")` - Specifies a comma-separated list of
+  blacklisted drivers . If specified, drivers in the blacklist will be
+  disabled. If the blacklist is empty, all drivers are fingerprinted and enabled
```

Comment:
I would remove the last sentences in both of these since it is pretty clear what the behavior will be and the fingerprinters/drivers still need to take into account the whitelist